### PR TITLE
Update integration tests to use REST APIs in master branch

### PR DESCRIPTION
**Pull request by ashanthamara**
## This PR will remove the soap based api implementations used in IS Integration Tests

- Added `RestBaseClient` for the common rest functions used in rest api
- Added `OAuth2RestClient` for application management rest api implementation 
- Added `SCIM2RestClient` for SCIM2 rest api implementation 
- Added `AuthenticatorRestClient` for authentication based rest api implementation
- Added `EmailTemplatesRestClient` for email template based rest api implementation
- Added `IdentityGovernanceRestClient` for identity-governance based rest api implementation
### OAuth2
- OAuth2RoleClaimTestCase 
- OAuth2DeviceFlowTestCase 
- OAuth2RequestObjectSignatureValidationTestCase 
- OAuth2IDTokenEncryptionTestCase 
- OAuth2ServiceErrorResponseTest 
- OAuth2ServiceRefreshTokenGrantTestCase 
- PermissionBasedScopeValidatorTestCase 
-  OAuth2ServiceClientCredentialTestCase 
- OAuth2ServiceSAML2BearerGrantTestCase 
- OAuth2ServiceImplicitGrantTestCase 
- OAuth2ServiceIntrospectionTestCase 
- OAuth2ServiceResourceOwnerTestCase 
- OAuth2TokenRevocationWithRevokedAccessToken
- OAuth2BackChannelLogoutTestCase 
- OAuth2TokenRevocationWithMultipleSessionTerminationTestCase 
- OAuth2TokenRevocationAfterAccountDisablingTestCase 
- Oauth2OPIframeTestCase 

### Identity/governance
- AccountLockEnabledTestCase 

## Related PRs sent to https://github.com/wso2/product-is/tree/7.x.x
- https://github.com/wso2/product-is/pull/15906
- https://github.com/wso2/product-is/pull/15931

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/EmailTemplatesRestClient.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        return headerList;
+    }
+
+    private String getEncodedEmailTemplateTypeId(String emailTemplateTypeId) {
+        return Base64.encodeBase64String(emailTemplateTypeId.getBytes());
+    }
+
+    /**
+     * Close the HTTP client.
+     *
```
[the remark this answers is not in the document]
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/EmailTemplatesRestClient.java]: Fixed with https://github.com/wso2/product-is/pull/16223/commits/3755cd5c2291840db84d83fb61b4d5fca9547f24

**Review by madurangasiriwardena — APPROVED** [review]

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java [hunk 2]
```diff
@@ -23,89 +23,103 @@
...
-        remoteUSMServiceClient.addRole(OAUTH_ROLE, null, null);
-        remoteUSMServiceClient.addUser(USERNAME, PASSWORD, null, getUserClaims(), "default",
-                false);
+        roleId = scim2RestClient.addRole(getRoleCreationInfo());
+        userId = scim2RestClient.createUser(getUserCreationInfo());
     }
 
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
-
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java:92]: Shall we fix the formatting issue by adding the missing new line?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RequestObjectSignatureValidationTestCase.java [hunk 3]
```diff
@@ -75,14 +76,16 @@ public void testInit() throws Exception {
 
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
-        deleteApplication();
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RequestObjectSignatureValidationTestCase.java:78]: Shall we fix the formatting issue by adding the missing new line?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RequestObjectSignatureValidationTestCase.java [hunk 4]
```diff
@@ -40,32 +42,31 @@
...
+import java.security.cert.Certificate;
 import java.security.interfaces.RSAPrivateKey;
 import java.util.Collections;
 import java.util.Date;
 import java.util.UUID;
 
-/*
+/**
     Integration tests for Signed Request Object validation.
  */
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RequestObjectSignatureValidationTestCase.java:61]: Shall we fix the formatting here.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java [hunk 5]
```diff
@@ -458,8 +464,8 @@ private void initServiceProviderKeys() throws Exception {
      * @param client         CloseableHttpClient object to send the login post.
      * @param sessionDataKey String sessionDataKey obtained.
      * @return Extracted sessionDataKeyConsent.
-     * @throws IOException
-     * @throws URISyntaxException
+     * @throws IOException Error
+     * @throws URISyntaxException Error
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java:468]: Missing fullstops at the end

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java [hunk 6]
```diff
@@ -491,8 +497,8 @@ private String getSessionDataKeyConsent(CloseableHttpClient client, String sessi
      * @param idToken  Encrypted ID token to be decrypted and checked.
      * @param audience Audience value that should be appeared in the token.
      * @return Boolean True if audience matches, False otherwise.
-     * @throws ParseException
-     * @throws JOSEException
+     * @throws ParseException Error
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java:500]: Missing fullstops at the end

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java [hunk 7]
```diff
@@ -431,7 +437,7 @@ private String getLocationHeaderValue(HttpResponse response) {
     /**
      * Initiate service provider keys required for the tests.
      *
-     * @throws Exception
+     * @throws Exception error
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java:440]: Missing fullstops

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java [hunk 8]
```diff
@@ -119,15 +123,17 @@ public void testInit() throws Exception {
 
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
-        deleteApplication();
-        removeOAuthApplicationData();
+        deleteApp(application.getId());
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java:126]: Shall we fix the formatting issue by adding the missing new line?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2DeviceFlowTestCase.java [hunk 9]
```diff
@@ -317,19 +328,34 @@ public HttpResponse sendGetRequest(HttpClient client, String locationURL) throws
     /**
      * Create Application with the given app configurations
      *
-     * @return OAuthConsumerAppDTO
-     * @throws Exception
+     * @return ApplicationResponseModel
+     * @throws Exception exception
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2DeviceFlowTestCase.java:332]: Shall we fix the formatting?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java [hunk 10]
```diff
@@ -310,40 +309,20 @@ private List<NameValuePair> getOIDCInitiationRequestParams() {
...
-        requestConfig.setInboundAuthType("oauth2");
-        if (StringUtils.isNotBlank(oidcAppClientSecret)) {
-            Property property = new Property();
-            property.setName("oauthConsumerSecret");
-            property.setValue(oidcAppClientSecret);
-            Property[] properties = {property};
-            requestConfig.setProperties(properties);
-        }
+    private void addAdminUser() throws Exception {
+        UserObject userInfo = new UserObject();
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:313]: Shall we add the missing new line?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java [hunk 11]
```diff
@@ -159,25 +145,38 @@ public void testOIDCLogout() throws IOException {
     }
 
     private void createOIDCApplication() throws Exception {
+        ApplicationModel application = new ApplicationModel();
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:148]: Shall we fix the formatting issue by adding the missing new line?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java [hunk 12]
```diff
@@ -172,112 +186,51 @@ public void testSendAuthorizedPostAfterRoleUpdate() throws Exception {
                 "Id token must not contain role claim which is not configured for the requested scope.");
     }
 
-    private ClaimValue[] getUserClaims() {
+    private UserObject getUserCreationInfo() {
+        UserObject userInfo = new UserObject();
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java:190]: Shall we fix the formatting issue by adding the missing new line?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java [hunk 13]
```diff
@@ -172,112 +186,51 @@ public void testSendAuthorizedPostAfterRoleUpdate() throws Exception {
...
-
-        if (authRequestList.size() > 0) {
-            serviceProvider.getInboundAuthenticationConfig()
-                    .setInboundAuthenticationRequestConfigs(authRequestList.toArray(new
-                            InboundAuthenticationRequestConfig[authRequestList.size()]));
-        }
-        appMgtclient.updateApplicationData(serviceProvider);
-        return appDtoResult;
+    private PatchOperationRequestObject getAddUserPatchRole(String userId) {
+        RoleItemAddGroupobj patchRoleItem = new RoleItemAddGroupobj();
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java:229]: Shall we fix the formatting issue by adding the missing new line?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java [hunk 14]
```diff
@@ -150,11 +179,72 @@ ServiceProvider setServiceProviderClaimConfig(ServiceProvider serviceProvider) {
     }
 
 	/**
-	 * Create Application with a given appDTO
+	 * Create Application with a given ApplicationModel
 	 *
-	 * @return OAuthConsumerAppDTO
-	 * @throws Exception
+	 * @param application application creation object
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java:184]: Shall fix the formatting issues by adding missing full stops? Let's fix other places as well.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java [hunk 15]
```diff
@@ -150,11 +179,72 @@ ServiceProvider setServiceProviderClaimConfig(ServiceProvider serviceProvider) {
...
+	 * @return application id
+	 * @throws Exception Exception
+	 */
+	public String addApplication(ApplicationModel application) throws Exception {
+		return restClient.createApplication(application);
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java:189]: Shall we fix the formatting issue by adding the missing new line? Let's fix other places as well.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java [hunk 16]
```diff
@@ -77,53 +74,48 @@ public static Object[][] configProvider() {
...
-        remoteUSMServiceClient = new RemoteUserStoreManagerServiceClient(backendURL, sessionCookie);
+        restClient = new OAuth2RestClient(serverURL, tenantInfo);
 
         setSystemproperties();
         client = HttpClientBuilder.create().build();
     }
 
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
-
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java:107]: Shall we add the missing new line here?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceRegexCallbackUrlTestCase.java [hunk 17]
```diff
@@ -224,4 +214,29 @@ public void testValidateAccessToken() throws Exception {
 		Assert.assertEquals(valid, "true", "Token Validation failed");
 		EntityUtils.consume(response.getEntity());
 	}
+
+	private ApplicationResponseModel createTestApplication() throws Exception {
+		ApplicationModel application = new ApplicationModel();
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceRegexCallbackUrlTestCase.java:219]: Shall we add the missing new line here?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithRevokedAccessToken.java [hunk 18]
```diff
@@ -237,9 +239,41 @@ private TokenIntrospectionResponse introspectAccessToken(AccessToken accessToken
         return TokenIntrospectionResponse.parse(introspectionHTTPResp);
     }
 
+    private ApplicationResponseModel createApp() throws Exception {
+        ApplicationModel application = new ApplicationModel();
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithRevokedAccessToken.java:243]: Shall we add the missing new line here?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/model/AssertionEncryptionConfiguration.java [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+     */
+    private String toIndentedString(Object o) {
+
+        if (o == null) {
+            return "null";
+        }
+        return o.toString();
+
+    }
+}
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/model/AssertionEncryptionConfiguration.java:137]: Shall we add the missing new line at the end of file?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java [hunk 20]
```diff
@@ -0,0 +1,232 @@
...
+
+    public SCIM2RestClient(String serverUrl, Tenant tenantInfo){
+        this.serverUrl = serverUrl;
+        this.tenantDomain = tenantInfo.getContextUser().getUserDomain();
+        this.username = tenantInfo.getContextUser().getUserName();
+        this.password = tenantInfo.getContextUser().getPassword();
+    }
+
+    /**
+     * Create a user
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java:61]: Shall we fix the formatting by adding missing full stops

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java [hunk 21]
```diff
@@ -0,0 +1,232 @@
...
+     *
+     * @param userInfo object with user creation details.
+     * @return Id of the created user.
+     */
+    public String createUser(UserObject userInfo) throws Exception {
+        String jsonRequest = toJSONString(userInfo);
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java:67]: Shall we add the missing new line?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/RestBaseClient.java [hunk 22]
```diff
@@ -0,0 +1,160 @@
...
+    }
+
+    /**
+     * To convert object to a json string.
+     *
+     * @param object Respective java object.
+     * @return Relevant json string.
+     */
+    public String toJSONString(java.lang.Object object) {
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/RestBaseClient.java:61]: Shall we fix the formatting issue by adding missing new line? Check other places as well

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/RestBaseClient.java [hunk 23]
```diff
@@ -0,0 +1,160 @@
...
+        JSONObject json = (JSONObject) parser.parse(responseString);
+        if (json == null) {
+            throw new Exception("Error occurred while getting the response");
+        }
+
+        return json;
+    }
+
+    /**
+     * Execute and get the response of HTTP POST
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/RestBaseClient.java:82]: Shall we add the missing fullstop. Check other places as well.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/IdentityGovernanceRestClient.java [hunk 24]
```diff
@@ -0,0 +1,96 @@
...
+    /**
+     * Update connector properties
+     *
+     * @param categoryId Connector category id.
+     * @param connectorId Connector id.
+     * @param connectorPatch Connector patch request object.
+     */
+    public void updateConnectors(String categoryId, String connectorId, ConnectorsPatchReq connectorPatch)
+            throws IOException {
+        String jsonRequest = toJSONString(connectorPatch);
```
Comment (UdeshAthukorala) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/IdentityGovernanceRestClient.java:70]: Shall we fix the formatting issue by adding missing new line?  Let's check other places as well

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java (hunk 2 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java:92]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RequestObjectSignatureValidationTestCase.java (hunk 3 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RequestObjectSignatureValidationTestCase.java:78]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java (hunk 5 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java:468]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RequestObjectSignatureValidationTestCase.java (hunk 4 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RequestObjectSignatureValidationTestCase.java:61]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java (hunk 6 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java:500]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java (hunk 21 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java:67]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/RestBaseClient.java (hunk 22 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/RestBaseClient.java:61]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/RestBaseClient.java (hunk 23 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/RestBaseClient.java:82]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/IdentityGovernanceRestClient.java (hunk 24 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/IdentityGovernanceRestClient.java:70]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java (hunk 7 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java:440]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java (hunk 8 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java:126]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java (hunk 10 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:313]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java (hunk 11 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2BackChannelLogoutTestCase.java:148]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java (hunk 12 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java:190]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java (hunk 13 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java:229]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java (hunk 15 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java:189]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java (hunk 14 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java:184]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java (hunk 16 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java:107]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceRegexCallbackUrlTestCase.java (hunk 17 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceRegexCallbackUrlTestCase.java:219]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/model/AssertionEncryptionConfiguration.java (hunk 19 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/model/AssertionEncryptionConfiguration.java:137]: Fixed with https://github.com/wso2/product-is/pull/20096

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2DeviceFlowTestCase.java (hunk 9 above)
Comment (ashanthamara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2DeviceFlowTestCase.java:332]: Fixed with https://github.com/wso2/product-is/pull/20096

**Pull request merged**